limbo/slack.py

```diff
@@ -68,9 +43,6 @@ def __init__(self, token):
         self.connected = False
```

Comment:
why does `slack.py` keep sets of `users` and `channels` at all? it doesn't seem like they're used anywhere...

do you have future plans for them? or am i just missing their use?

Reply:
No, I mainly only kept them around because they're part of the API so people may be relying on them in their own plugins. I don't have any that use them either.